include/gmsh.tcc

```diff
@@ -225,15 +225,17 @@ void GMSH<Tdim, Tvertices>::compute_material_points(unsigned ngauss_points) {
 
   Eigen::VectorXd pointsarray(Tdim);
 
-  //! last_global_id should be changed later if more than one material
+  //! initial_points_id should be changed later if more than one material
```

Comment:
`initial_points_id` doesn't describe what is supposed to do, especially when it is incremented later.